app/models/middleware_server.rb

```diff
@@ -12,4 +12,22 @@ class MiddlewareServer < ApplicationRecord
   def metrics_capture
     @metrics_capture ||= ManageIQ::Providers::Hawkular::MiddlewareManager::LiveMetricsCapture.new(self)
   end
+
+  def tenant_identity
```

Comment:
Can you please explain a bit more about this method?
in which case there's no `ext_management_system`? 
I'm not sure I understand where the `ext_management_system` variable is taken from (and why is it the base class for all providers) or how can it be that there is no provider?
thanks!